code/popup.html

```diff
@@ -31,9 +31,11 @@
             </div>
 
             
-            <div class="col-sm-4">
-
-                  <a class="btn float-right" target="_blank" id="help" href="javascript:void(0)">Help</a>
+            <div class="col-sm-5">
+				
+				  <a class="btn float-right" target="_blank" title="Works only when you are online" href="https://codecompilerun.herokuapp.com/">OnlineComplie</a>
```

Comment:
Looks like you have a typo in the link text.

Reply:
I don't found any typo error. Please check it once again.
still if you find it , please tell in detail.

Reply:
Online _Complie_

Reply:
Ohk I will correct it. 😃

Reply:
I have changed the typo error.please verify it.